puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -73,6 +73,11 @@
   $n1kv_vsm_ip                   = undef,
   $n1kv_vsm_password             = undef,
   $n1kv_vsm_username             = undef,
+  $plumgrid_director_vip         = undef,
```

Comment:
I have added these parameter in quickstack::pacemaker::neutron. 
plumgrid_director_vip is not pacemaker vip rather created externally by plumgrid classes/packaging itself through keepalived.